src/Symfony/Bundle/SecurityBundle/Resources/config/security_debug.xml

```diff
@@ -10,5 +10,11 @@
         <service id="debug.security.access.decision_manager" class="Symfony\Component\Security\Core\Authorization\TraceableAccessDecisionManager" decorates="security.access.decision_manager">
             <argument type="service" id="debug.security.access.decision_manager.inner" />
         </service>
+
+        <service id="debug.security.firewall" class="Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener" decorates="security.firewall" public="true">
```

Comment:
this service should not be public. There is no reason to do a `$container->get('debug.security.firewall')